src/tir/transforms/storage_rewrite.cc

```diff
@@ -1755,7 +1793,7 @@ Pass StorageRewrite() {
     // padded out to 32 bits) would require either rewriting
     // AllocateConst::data, or would require the code generators to
     // handle vectorized constants.
-    return PointerValueTypeRewrite(std::move(f), true, false, false, true, true, true, false,
+    return PointerValueTypeRewrite(std::move(f), true, false, false, false, true, true, false,
```

Comment:
The fourth condition must be false, as the vectorized buffer merge (for example, merge half B_shared[1024] into halfx8 B_shared[128]) will occasional lead to a unhandled behavior during async copy lowering phase.